hyperspy/_components/gaussian2d.py

```diff
@@ -90,3 +117,12 @@ def fwhm_y(self):
     @fwhm_y.setter
     def fwhm_y(self, value):
         self.sigma_y.value = value / sigma2fwhm
+
+    @property
+    def rotation_degrees(self):
+        """ Angle between major axis and x-axis."""
+        rotation = math.fmod(self.rotation.value, pi2) 
```

Comment:
returning to my comment in #864 - is it not constrained to between 0 and pi, since the semi-major axes are indistinguishable?